mino/option.go

```diff
@@ -30,6 +30,25 @@ func ApplyFilters(filters []FilterUpdater) *Filter {
 // FilterUpdater is a function to update the filters.
 type FilterUpdater func(*Filter)
 
+// RotateFilter is a filter to rotate the indices. When n is above zero, it will
```

Comment:
so we should update the comment at line 12: "This list is always sorted."

Reply:
There's already the comment ;)